src/utils/createObjectMethodTransform.js

```diff
@@ -41,6 +41,20 @@ export default function createObjectMethodTransform(polyfill) {
         return
       }
 
+      if (
+        t.isMemberExpression(path.node.callee) &&
+        t.isIdentifier(path.node.callee.property) &&
+        t.isMemberExpression(path.node.arguments[0]) &&
+        path.node.arguments[0].object.name === object &&
+        path.node.arguments[0].property.name === property
```

Comment:
We can't assume it is the first argument, we have to check all arguments.

```js
function ourSortAlgorithm(arr, signFn) {
	return arr.slice().sort((a, b) => signFn(a - b))
}

// This
ourSortAlgorithm([1, 2, 3], Math.sign)

// Will be replaced with
ourSortAlgorithm([1, 2, 3], Math.sign)

// instead of this because it is the second argument
ourSortAlgorithm([1, 2, 3], _Math_sign)
```